src/components/Template/Tocbot/index.js

```diff
@@ -17,7 +17,7 @@ const tocbot = (typeof window !== 'undefined')
 export default class Tocbot extends React.Component {
   componentDidMount () {
     if (tocbot) {
-      tocbot.init(TOCBOT_OPTIONS)
+      tocbot.init(Object.assign({}, this.props, TOCBOT_OPTIONS))
```

Comment:
I think you want `Object.assign({}, TOCBOT_OPTIONS, this.props)`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign